pkg/controller/kogitoapp/resource/deployment_config.go

```diff
@@ -48,7 +50,12 @@ const (
 	envVarKafkaBootstrapURI    = "KAFKA_BOOTSTRAP_SERVERS"
 	envVarKafkaBootstrapSuffix = "_BOOTSTRAP_SERVERS"
 
-	envVarExternalURL = "KOGITO_SERVICE_URL"
+	envVarExternalURL        = "KOGITO_SERVICE_URL"
+	downwardAPIVolumeName    = "podinfo"
+	downwardAPIVolumeMount   = kogitoHome + "/" + downwardAPIVolumeName
```

Comment:
default location for downward api lives on /var, why defaulting to kogito home?
maybe this and the next references to the downward api needs to be renamed to reflect the real purpose of this.

Reply:
Looking at the docs, I haven't found any mentions to a default to `/var`:
https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/

Where can I find this info? I just set there because it is the default path for all purposes of the service. I don't mind changing to `/var` if it's the default. It's pretty simple to do that.

Reply:
nevermind, I got confused with the another info available on /var, i.e. ca.crt, namespace, etc.
: